packages/plugins/type-designer/src/components/drawer-content.svelte

```diff
@@ -0,0 +1,78 @@
+<script lang="ts">
+// CONSTANTS
+import { SCD_ELEMENTS } from '@oscd-plugins/core'
+// SMUI COMPONENTS
+import Textfield from '@smui/textfield'
+import Button from '@smui/button'
+// STORES
+import { pluginStore } from '@/stores/plugin.store'
+import { drawerStore } from '@oscd-plugins/ui'
+// TYPES
+import type { DataTypeTemplates } from '@oscd-plugins/core'
+
+//====== INITIALIZATION ====//
+
+// props
+export let typeElement: DataTypeTemplates.TypeElement
+export let currentColumn: keyof typeof SCD_ELEMENTS
+
+const { element, ...attributes } = typeElement
+
+//====== REACTIVITY ====//
+
+$: standardAttributes = SCD_ELEMENTS[currentColumn]?.element.standardAttributes
+$: localAttributes = attributes
+
+//====== FUNCTIONS =====//
+
+function updateTypeElement() {
+	pluginStore.createAndDispatchUpdateActionEvent({
+		element,
+		oldAttributes: attributes,
+		newAttributes: localAttributes
+	})
+	drawerStore.closeDrawer()
+}
+</script>
+
+<div id="type-designer-drawer">
+	<div class="form">
+		{#each standardAttributes as attribute}
+			<Textfield class="textField" variant="outlined" bind:value={localAttributes[attribute]} label={attribute} />
+		{/each}
+	</div>
+	<div class="action">
+		<Button on:click={drawerStore.closeDrawer}>Cancel</Button>
```

Comment:
I find it better to have `drawerStore.closeDrawer` as a handlerCloseDrawer function

Reply:
Refactored see 584f1fc